bin/install-elixir.sh

```diff
@@ -70,5 +70,5 @@ rm elixir.zip
 # it may be called without a preceding configure call, for instance when
 # building packages from a dist tarball. So we ensure it has hex there already.
 echo "===> Installing Hex"
-MIX_HOME=/home/jenkins/.mix /usr/local/bin/mix local.hex --force
-chown -R jenkins:jenkins /home/jenkins
+MIX_HOME=${HOME}.mix /usr/local/bin/mix local.hex --force
+chown -R $(whoami):$(whoami) ${HOME}
```

Comment:
I am not sure if this was running as jenkins or as root. It might have been as root then whoami, won't work. Maybe a CI_USER or similar variable might work here.

This might not be an issue any longer after https://github.com/apache/couchdb/pull/5250. If this gets a bit tricky to fix we can probably just skip installing hex altogether and let `make clean || true` do it's thing on failure of `make clean`.